csidriver/cmd/csidriver.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/kubeedge/kubeedge/csidriver/pkg/app"
+)
+
+func init() {
+	flag.Set("logtostderr", "true")
+}
+
+var (
+	endpoint   = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
+	driverName = flag.String("drivername", "csidriver", "name of the driver")
+	nodeID     = flag.String("nodeid", "", "node id")
+	keEndpoint = flag.String("kubeedgeendpoint", "unix:///kubeedge/kubeedge.sock", "kubeedge endpoint")
+	version    = flag.String("version", "", "version")
+)
+
+func main() {
+	flag.Parse()
```

Comment:
suggest to use cobra and pflag, like edgecore,cloudcore

Reply:
Thanks. Perhaps we can improve it later:)

Reply:
Done. Thanks @kadisi